crypto/ct/ct_b64.c

```diff
@@ -115,32 +115,38 @@ SCT *SCT_new_from_base64(unsigned char version, const char *logid_base64,
     return NULL;
 }
 
-CTLOG *CTLOG_new_from_base64(const char *pkey_base64, const char *name)
+int CTLOG_new_from_base64(CTLOG **ct_log, const char *pkey_base64, const char *name)
 {
     unsigned char *pkey_der = NULL;
     int pkey_der_len = ct_base64_decode(pkey_base64, &pkey_der);
     const unsigned char *p;
     EVP_PKEY *pkey = NULL;
     CTLOG *log = NULL;
 
+    if (ct_log != NULL)
+        *ct_log = NULL;
+
     if (pkey_der_len <= 0) {
         CTerr(CT_F_CTLOG_NEW_FROM_BASE64, CT_R_LOG_CONF_INVALID_KEY);
-        return NULL;
+        return 0;
     }
 
     p = pkey_der;
     pkey = d2i_PUBKEY(NULL, &p, pkey_der_len);
     OPENSSL_free(pkey_der);
     if (pkey == NULL) {
         CTerr(CT_F_CTLOG_NEW_FROM_BASE64, CT_R_LOG_CONF_INVALID_KEY);
-        return NULL;
+        return 0;
     }
 
     log = CTLOG_new(pkey, name);
     if (log == NULL) {
         EVP_PKEY_free(pkey);
-        return NULL;
+        return -1;
```

Comment:
I think the return values need documenting somewhere - probably as a comment before the start of the function. I note that this *new* function doesn't have an associated pod file...or any of the other CTLOG functions for that matter....grrrrr....oh well, not within the scope of this PR. :-(